airflow/www/utils.py

```diff
@@ -282,7 +289,18 @@ def dt(attr):
         f = f.isoformat() if f else ''
         if timezone.utcnow().isoformat()[:4] == f[:4]:
             f = f[5:]
-        return Markup("<nobr>{}</nobr>").format(f)
+        return Markup('<time datetime="{}">{}</time>').format(f, f)
+    return dt
+
+
+def localized_f(attr_name):
+    def dt(attr):
+        f = attr.get(attr_name)
+        if f is None:
+            return Markup('<time datetime="{}">{}</time>').format(f, f)
```

Comment:
This would yield `<time datetime="None">None</time>` which would break moment.js wouldn't it?. I guess we probably want 
```suggestion
            return None
```

or `return ''`?